testdata/terraform/v5/cloudflare_account/test.tf

```
@@ -1,7 +1,23 @@
-resource "cloudflare_account" "terraform_managed_resource" {
-  name = "name"
-  type = "standard"
-  unit = {
-    id = "f267e341f3dd4697bd3b9f71dd96247f"
+resource "cloudflare_account" "terraform_managed_resource_0" {
```

```suggestion
resource "cloudflare_account" "terraform_managed_resource" {
```

the internals should be making this correctly trimmed for you (if `CI` is set).

---

@jacobbednarz Not sure if I follow. The response had 3 items in it, so from what i understood, wouldn't the logic enumerate them? 
I can update the api yaml to have only one item in the reponse.

---

i don't recall that being a thing. i thought it blindly trimmed it and we needed to manually rename it for the assertion. good to add a test for that scenario either way.

---

This was a change I made to allow the tf validation to pass. If we want to revert that to force manual renaming, I can revert it

---

Ah I see. If we need to revert it i can do that in this change. @musa-cf @jacobbednarz